packages/core/content-type-builder/server/src/services/architect.ts

```diff
@@ -151,12 +152,46 @@ const systemPrompt = `
           </examples>
     `;
 
+const applyDefaultValues = (schema: any) => {
+  // Transform info properties to be kebab-case always
+  schema.info.singularName = schema.info.singularName.toLowerCase().replace(/\s+/g, '-');
+  schema.info.pluralName = schema.info.pluralName.toLowerCase().replace(/\s+/g, '-');
+  if (!schema.info.name) schema.info.name = schema.info.pluralName;
+
+  // Apply default values to attributes
+  Object.entries(schema.attributes).forEach(([name, attribute]: any) => {
+    switch (attribute.type) {
+      case 'integer':
+        break;
+      case 'email':
+        break;
+      case 'boolean':
+        break;
+      case 'enumeration':
+        schema.attributes[name].enum = ['a', 'b'];
+        break;
+      case 'relation':
+        schema.attributes[name].relation = 'oneToOne';
+        schema.attributes[name].target = 'api::tag.tag';
+        break;
+      case 'media':
+        schema.attributes[name].allowedTypes = ['images'];
+        schema.attributes[name].multiple = false;
+        break;
+      default:
+        break;
+    }
+  });
+
+  return schema;
+};
+
 export const create = async (prompt: string, previousSchema?: object) => {
   console.log({ previousSchema });
 
   const { object } = await generateObject({
     model: anthropic('claude-3-sonnet-20240229', {}),
-    maxRetries: 3,
+    maxRetries: 10,
```

Comment:
We should probably make this configurable :sweat_smile:

Reply:
I agree could cast it to process.env

Reply:
sure!